### PR TITLE
feat(web): hippy-reat-web添加HippyEventListener以及HippyEventEmitter两个类，实现事件监听功能

### PR DESCRIPTION
**Pull request by STU-Mike**
Before submitting a new pull request, please make sure:

- [ ] Test cases have been added/updated for the code you will submit.
- [ ] Documentation has added or updated.
- [ ] Commit message is following the [Convention Commit](https://conventionalcommits.org/) guideline.
- [ ] Squash the repeat code commits, short patches are welcome.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=323) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=323) before we can accept your contribution.<br/><hr/>**mikejwmai** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/Hippy?pullRequest=323) it.</sub>

File: packages/hippy-react-web/src/events/listener.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import HippyEventEmitter from './emitter';
+
+const ee = require('event-emitter');
```
Comment (xuqingkuang) [inline packages/hippy-react-web/src/events/listener.ts:3]: 这里不要用 require，两种 module 混用会出问题

File: packages/hippy-react-web/src/events/index.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export { default as HippyEventEmitter } from './emitter';
```
Comment (xuqingkuang) [inline packages/hippy-react-web/src/events/index.ts:1]: 这个文件其实不需要，直接在最外层 index.js 里引用吧。

File: packages/hippy-react-web/src/events/listener.ts [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  }
+
+  // 取消事件的注册
+  public unregister() {
+    // 直接清空回调函数
+    allOff(this.evenInstance);
+    // 初始化序列号
+    this.nextIdForHandler = 0;
+    // 清空回调函数
+    // this.handlerContainer.forEach((value, key) => {
```
Comment (xuqingkuang) [inline packages/hippy-react-web/src/events/listener.ts:80]: 这类代码注释全部删掉

**Review by chr688 — APPROVED** [review]

**Comment by chuyik** [comment]
ping @STU-Mike @xuqingkuang

**Comment by zoomchan-cxj** [comment]
hippy-reat-web will be decpreacated soon, hippy-web-renderer will be used

**Pull request closed without merging**